### DISCUSSION TITLE
FF 开发者中心 "/version" should match format "versionString"

### DISCUSSION THREAD

**Discussion by ted423** [ronggang/PT-Plugin-Plus#882] [Q&A] [answered]
https://github.com/mozilla/addons-linter/issues/1905
似乎version不能写1.5.2.d174e1a
其实我没看懂...写1.5.2.20210825也不行，我就写了个1.5.2.1

**Comment by Rhilip** [comment, marked as answer]
不是很懂，哪里出了差错？
version本来就是 `1.5.2`，`1.5.2.d174e1a` 是 version_name 。

refs: 
 - https://developer.mozilla.org/zh-CN/docs/Mozilla/Add-ons/WebExtensions/manifest.json/version
 - https://developer.chrome.com/docs/extensions/mv3/manifest/version/
